test/release/butternut.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+alias uglify-js=$PWD/bin/uglifyjs
+rm -rf tmp/butternut
+git clone https://github.com/Rich-Harris/butternut.git tmp/butternut
+cd tmp/butternut
+rm -rf .git/hooks
+patch -l -p1 <<EOF
+--- a/package.json
++++ b/package.json
+@@ -25 +24,0 @@
+-    "prepublish": "npm run test:min",
+--- a/rollup.config.js
++++ b/rollup.config.js
+@@ -1 +0,0 @@
+-import buble from 'rollup-plugin-buble';
+@@ -28,6 +26,0 @@ const config = {
+-               buble({
+-                       include: ['src/**', 'node_modules/acorn/**'],
+-                       transforms: {
+-                               dangerousForOf: true
+-                       }
+-               }),
+EOF
+UGLIFY_OPTIONS=$@
+
+function minify_in_situ() {
```

Comment:
https://github.com/mishoo/UglifyJS/runs/2044324285?check_suite_focus=true#step:3:61
```
Cloning into 'tmp/butternut'...
patching file package.json
patching file rollup.config.js
./test/release/butternut.sh: 27: Syntax error: "(" unexpected
```

Reply:
Dropping the `function` keyword seems to resolve it.

Reply:
Ah - may have been a bash-ism.